content/posts/fas-verification-automation.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Automation of FAS verification in Packit Service"
+date: 2022-05-24
+weight: 94
+---
+
+As you may already know, for using Packit Service
+Github App we [require our users to have a valid Fedora Account System account](/docs/packit-service/#requirements-for-running-packit-service-jobs).
+We were verifying the newcomers until now manually, but in recent weeks, we have implemented an automated solution
+for it. Let's take a closer look at how it is done currently and what have we improved!
+
+Formerly, the process of verification by us started by waiting for the users to provide
+us their FAS username, then checking whether the provided FAS account
+exists and matches, and finally, manually adding the account to our allowlist in the database. For the communication with
+new users, we have used our [`packit/notifications`](https://github.com/packit/notifications/issues)
+repository on GitHub where we created an issue for each new installation.
+
+Although in general, this worked, it required human interaction and since we are not available 24/7, the verification wasn't immediate.
+We wanted to simplify the process for both users and us.
+Since in FAS, everyone can set their GitHub login that is then publicly available, we decided to utilize this setting.
+
+So how does the verification work now?
+
+For each new GitHub installation, we first check whether there isn't a FAS account with the same login as the one
+that triggered the installation. If we find such an account, we check whether the `GitHub Username` in this FAS account matches
+the Github login of the one that triggered the installation. To get the information about the FAS accounts, we
+use the [`fasjson-client` library](https://github.com/fedora-infra/fasjson-client).
+If this check doesn't prove any match, we create an issue in the
+`packit/notifications` repository as previously. This is what it looks like:
+
+![Issue in packit/notifications](/images/notifications-repo-issue.png)
```

Comment:
What about removing the header so we have a white background and the issue text is better readable?

Reply:
firstly I wanted to emphasize that the issue is in packit/notifications repo, but it is described in the text well, so you are right, it looks better without it, thanks!